src/content/docs/build/applications/add-and-manage-applications.mdx

```diff
@@ -16,12 +16,10 @@ Applications are how you make your product or service available to your users, s
 
 If you plan to manage authentication for each of your applications using Kinde, you will need to add and configure each application.
 
-[Kinde provides several application types](/build/applications/about-applications/), including a regular web app (for execution on a server), single page web app (for browsers and mobile), and machine to machine applications (for connecting to Kinde’s API).
+[Kinde provides several application types](/build/applications/about-applications/), including a regular web app (for execution on a server), single page web app (for browsers and mobile), and machine to machine applications (for backend services that require access to an API, including the [Kinde management API](/developer-tools/kinde-api/connect-securely-to-kinde-api/)).
```

Comment:
Hey @clairekinde11 
This link `/developer-tools/kinde-api/connect-securely-to-kinde-api/` is invalid, we need to add the new one instead.